[BUGFIX] Corriger le docker-compose.yml cassé sur pix-site (PIX-14301)

lego-technix opened the pull request:
## :unicorn: Problème

Actuellement sur les postes de développement on ne peut plus exécuter _pix-site_ derrière nginx dans un conteneur Docker.

Or cette possibilité est très importante car elle permet de pouvoir développer en local en étant dans des conditions au plus près de la production et sans devoir déployer à répétition sur Scalingo, ce qui est extrêmement long, peu productif et très consommateur en ressources (systèmes, environnementales).

Actuellement l'erreur la suivante lorsqu'on exécute la commande `npm run start:nginx` : 
```shell
npm run start:nginx
…
production_nginx | 2024/09/13 09:53:27 [emerg] 1#1: open() "/app/nginx/includes/rewrites.conf" failed (2: No such file or directory) in /etc/nginx/conf.d/default.conf:57
```

## :robot: Proposition

Dans `docker-compose.yml` ajouter le volume manquant `/app/nginx/includes/rewrites.conf` qui correspond à un fichier récemment ajouté dans `servers.conf.erb` mais pas mis à jour dans la configuration Docker pour le développement en local.

## :rainbow: Remarques

Un fichier `docker-compose.yml` à la racine du dépôt a également été supprimé car il n'est plus utilisé. C'est un reste de l'ancienne architecture de ce dépôt.

## :100: Pour tester

:warning: Avant de tester en local, vérifier qu'aucun serveur web (nginx, Apache, etc.) ou autre service n'est démarré et n'écoute sur le port `80`.

1. Exécuter les commandes suivantes en local et vérifier que pix-site est bien disponible en local sur le port `80` : 
```shell
npm run generate
npm run start:nginx
```

2. Vérifier que la modification du fichier `pix-site/docker-compose.yml` et la suppression du fichier ` ./docker-compose.yml` n'ont eu aucun impact sur les RA Scalingo qui n'en dépendent pas.

lego-technix committed:
fix(pix-site): fix broken docker-compose.yml

by adding the missing volume /app/nginx/includes/rewrites.conf used in servers.conf.erb

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [Pix Site (.fr)](https://site-pr709.review.pix.fr)
  * [Pix Site (.org)](https://site-pr709.review.pix.org)
  * [Pix Pro (.fr)](https://pro-pr709.review.pix.fr)
  * [Pix Pro (.org)](https://pro-pr709.review.pix.org)

Les variables d'environnement seront accessibles sur scalingo https://dashboard.scalingo.com/apps/osc-fr1/pix-site-review-pr709/environment

lego-technix committed:
refactor: remove now useless root docker-compose.yml

because the docker-compose.yml files need to be in pix-site and pix-pro.

The docker-compose.yml at the root of the repository is a leftover of the
previous architecture of the repository.

matthiasferraina reviewed (approved):
👌🏻
